### ISSUE TITLE
Prevent file duplication in split editor mode

### ISSUE THREAD

**Issue by DanielSchranc** [Microsoft/vscode#17377]
- VSCode Version: 1.7.2
- OS Version: Windows 10

Steps to Reproduce:

1. open files in one editor
2. hit 'ctrl+\'
3. last file in editor gets duplicated and opens in both split editor windows

Is it possible to overwrite this behavior in user settings.json?

**Comment by kieferrm** [comment]
@DanielSchranc what would you expect to happen?

**Comment by DanielSchranc** [comment]
@kieferrm @bpasero Well, I would expect it to not copying files into new split view, just split the view and let user to drag and drop whichever file to the new split view. Because how it works now is sometimes a bit confusing.
<img width="1439" alt="screen shot 2016-12-17 at 10 02 17" src="https://cloud.githubusercontent.com/assets/13023383/21285688/1be13766-c440-11e6-8efd-5c6f177f1d9b.png">

**Comment by bpasero** [comment]
@DanielSchranc for that to work we need https://github.com/Microsoft/vscode/issues/15405

**Comment by TKAB** [comment]
#15405 would only be required if only one file is opened in the current editor group. Why don't we let the behavior stay the same for the single file case and more than one opened editor in the current editor group the active editor moves to the newly opened editor group?

**Comment by ProLoser** [comment]
I agree with @TKAB completely. This is a duplicate of #8888 and it just got closed again. Is there any other duplicate issues that this keeps popping up with? Can we at least make this configurable?
